app/assets/stylesheets/components/_class-card.scss

```diff
@@ -0,0 +1,37 @@
+$class-card-padding-y: $small-spacing;
+$class-card-padding-2y: $base-spacing;
+
+.class-card {
+  @include margin($base-spacing null);
+  border: $thickest-border-width solid $black-haze;
+  padding: $class-card-padding-2y;
```

Comment:
Ah yes. That was the intention, but I overlooked the declaration on line 8. Perhaps it's too much for such a small component?